### PR TITLE
Add K8S Client 2.5.2.wso2v1

### PR DESCRIPTION
**Pull request by chamilad**
This PR contains the orbit bundle for Fabric8 Kubernetes Client 2.5.2 release.

### CODE REVIEW

File: kubernetes-client/2.5.2.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    <artifactId>kubernetes-client</artifactId>
+    <version>2.5.2.wso2v1</version>
+    <packaging>bundle</packaging>
+    <name>kubernetes-client.wso2</name>
+    <description>
+        This bundle exports io.fabric8.kubernetes.client
+    </description>
+    <url>http://wso2.org</url>
+
+    <repositories>
```
Comment (thusithathilina) [inline kubernetes-client/2.5.2.wso2v1/pom.xml]: Do we need this?

File: kubernetes-client/2.5.2.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline kubernetes-client/2.5.2.wso2v1/pom.xml]: Lince header should be 2017

Comment (chamilad) [inline kubernetes-client/2.5.2.wso2v1/pom.xml]: Fixed in 0dfb3d5.

File: kubernetes-client/2.5.2.wso2v1/pom.xml (hunk 1 above)
Comment (chamilad) [inline kubernetes-client/2.5.2.wso2v1/pom.xml]: Looks like it's not. We are not including any WSO2 artifacts. Removed in 0dfb3d5.

**Pull request merged**